public/docs/ts/latest/guide/router.jade

```diff
@@ -1768,47 +1869,46 @@ a#fragment
   ### Lazy-Loading route configuration
 
-  We'll start by pulling our `redirect` and `crisis-center` routes out of our `CrisisCenterModule` and including them in our
-  `app.routing.ts` file. We want to load our `Crisis Center` asynchronously, so we'll use the `loadChildren` property in
-  our route config where previously we used the `children` property to include our child routes.
+  We'll start by adding an `admin` route to our `app.routing.ts` file. We want to load our `Admin` module asynchronously,
+  so we'll use the `loadChildren` property in our route config where previously we used the `children` property to include our child routes.
 
-  We'll also change our `crisis-center` **path** in our `crisis-center.routing.ts` to an empty path. The `Router` supports
+  We'll also change our `admin` **path** in our `admin.routing.ts` to an empty path. The `Router` supports
   *empty path* routes, which we can use for grouping routes together without adding anything additional paths to the URL. Our
-  users will still visit `/crisis-center` and our `CrisisCenterComponent` still serves as our *Routing Component* which contains
+  users will still visit `/admin` and our `AdminComponent` still serves as our *Routing Component* which contains
   our child routes.
 
 +makeTabs(
-  `router/ts/app/app.routing.6.ts,
-  router/ts/app/crisis-center/crisis-center.routing.ts`,
-  'lazy-load-crisis-center,lazy-load-crisis-center',
+  `router/ts/app/app.routing.ts,
+  router/ts/app/admin/admin.routing.ts`,
+  'lazy-load-admin,',
   `app.routing.ts (load children),
-  crisis-center.routing.ts (empty path crisis center)
+  app/admin/admin.routing.ts (empty path admin)
   `)
 
 :marked
-  The `loadChildren` property is used by the `Router` to map to our bundle we want to lazy-load, in this case being the `CrisisCenterModule`.
+  The `loadChildren` property is used by the `Router` to map to our bundle we want to lazy-load, in this case being the `AdminModule`.
 
-  If we look closer at the `loadChildren` string, we can see that it maps directly to our `crisis-center.module` file where we previously built
-  out our `Crisis Center` feature area. After the path to the file we use a `#` to denote where our file path ends and to tell the `Router` the name
-  of our `CrisisCenter` NgModule. If we look in our `crisis-center.module` file, we can see it matches name of our exported NgModule class.
+  If we look closer at the `loadChildren` string, we can see that it maps directly to our `admin.module.ts` file where we previously built
+  out our `Admin` feature area. After the path to the file we use a `#` to denote where our file path ends and to tell the `Router` the name
+  of our `AdminModule`. If we look in our `admin.module.ts` file, we can see it matches name of our exported module class.
 
-+makeExcerpt('app/crisis-center/crisis-center.module.ts (export)', 'crisis-center-module-export')
++makeExcerpt('app/admin/admin.module.ts (export)', 'admin-module-export')
```

Comment:
@johnpapa Apparently the spread operator is still required in routes. I tried removing them all and it wouldn't compile.

We are NOT showing `export default` because we can't make it work with AoT (yet).